cli/args/mod.rs

```diff
@@ -1766,6 +1766,9 @@ fn resolve_node_modules_folder(
     .and_then(|c| c.specifier.to_file_path().ok())
   {
     config_path.parent().unwrap().join("node_modules")
+  } else if let Ok(init_cwd) = std::env::var("INIT_CWD") {
+    // TODO(nathanwhit): figure out a better solution
+    PathBuf::from(init_cwd).join("node_modules")
```

Comment:
Might cause stuff to break when using deno_task

Reply:
Yeah that was my concern too. There's definitely a better way to fix this, but couldn't immediately think of one